README.md

```diff
@@ -1,29 +1,48 @@
 # YJDH services
 
-## kesaseteli
+## kesaseteli employer
 
-YJDH-Kesäseteli
+YJDH-Kesäseteli service for employers to fulfill employee applications
 
-### kesaseteli Development with Docker
+### kesaseteli-employer Development with Docker
 
-1. Copy the contents of `.env.kesaseteli.example` to `.env.kesaseteli` and modify it if needed.
+1. Run `yarn` to install necessary packages
 
-2. Run `yarn kesaseteli` or, if you want to rebuild, then `yarn kesaseteli --build`
+2. Copy the contents of `.env.kesaseteli.example` to `.env.kesaseteli` and modify it if needed.
+
+3. Run `yarn employer` or, if you want to rebuild, then `yarn employer --build`
 
 The Frontend is now running at [localhost:3000](https://localhost:3000)
 The backend is now running at [localhost:8000](https://localhost:8000)
```

Comment:
Similar thing as in https://github.com/City-of-Helsinki/yjdh/pull/542/commits/6802d328a8ed55fa34b7f53ad8303d1d0213a5c7#r767705181